requirements/base.txt

```diff
@@ -9,7 +9,7 @@ django-mptt==0.9.0
 djangorestframework-csv==2.0.0
 tqdm==4.19.5                     # progress bars
 requests==2.18.4
-cherrypy==13.0.0
+cherrypy==11.0.0  # pyup: <11.1.0
```

Comment:
Looks good, but add an additional comment about why it has been changed. For instance: `# Temporarily pinning this until CherryPy stops depending on namespaced package, see #1234  # pyup: <11.1.0`

Reply:
Thank you! I'm going to add this comment now